context/app/static/js/pages/Vis/Vis.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useContext } from 'react';
+import Paper from '@material-ui/core/Paper';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+
+import { StyledTableContainer, HeaderCell } from 'js/shared-styles/Table';
+import { AppContext } from 'js/components/Providers';
+import useSearchData from 'js/hooks/useSearchData';
+
+/* JSON query
+ {
+     "size": 0,
+     "query": {
+         "bool": {
+             "filter": {
+                 "term": {
+                     "entity_type": "donor"
+                 }
+             }
+         }
+     },
+     "aggs": {
+         "mapped_metadata.sex": {
+             "terms": {
+                 "field": "mapped_metadata.sex.keyword"
+             }
+         }
+     }
+ }
+
+ */
+
+const donorSexQuery = {
+  size: 0,
+  query: {
+    bool: {
+      filter: {
+        term: {
+          entity_type: 'donor',
+        },
+      },
+    },
+  },
+  aggs: {
+    'mapped_metadata.race': {
+      terms: {
+        field: 'mapped_metadata.race.keyword',
+      },
+    },
+  },
+};
+
+// change to match the data
+const columns = [
+  { id: 'whie', label: 'White' },
+  { id: 'black', label: 'Black or African American' },
+];
+
+function Vis() {
+  const { elasticsearchEndpoint, nexusToken } = useContext(AppContext);
+  // eslint-disable-next-line no-unused-vars
```

Comment:
```suggestion
```
Maybe searchData wasn't used in an earlier version? shouldn't need to disable the warning now.